spec/models/section_spec.rb

```diff
@@ -1,17 +1,33 @@
 require "rails_helper"
 
 RSpec.describe Section do
-  context 'when a section is updated' do
-    context 'when period information is updated' do
+    context 'when a section is updated' do
+      context 'notify Kafka topic' do
+         let(:param) { rand }
+         subject(:responder) { ::SectionsResponder.new }
+
+           describe '#send_notification' do
+                 it 'executes call method with self parameter' do
+                     expect(responder.call(self))

```

Comment:
I don't think this is actually testing anything. it looks like this is just expecting the return value of `responder.call(self)` to be truthy. See: https://stackoverflow.com/questions/16677718/how-to-test-models-callback-method-independently

Reply:
In general for unit tests do we add or do not add a test that outputs a value we expect? I just want to understand the best way to test moving forward.

Reply:
It depends. If we are testing an individual method (which is often the case), then yes. That's not the case here though. Here we want to test that when a certain action (updating a section) occurs, a certain method (`SectionsResponder#call`) is called. Such tests are generally less desirable, but are often necessary when testing event-driven behavior that heavily intertwined with a framework.